symphony-bdk-http/symphony-bdk-http-webclient/src/main/java/com/symphony/bdk/http/webclient/ApiClientWebClient.java

```diff
@@ -137,47 +138,61 @@ public ApiClientWebClient(final WebClient webClient, String basePath, Map<String
       requestBodySpec.body(BodyInserters.fromValue(body));
     }
 
-    Mono<ClientResponse> mono = requestBodySpec.exchange();
-    ClientResponse response = mono.block();
+    try {
+      return requestBodySpec.exchangeToMono(response -> toApiResponse(returnType, response))
+          .block();
+    } catch (Exception e) {
+      Throwable unwrap = Exceptions.unwrap(e);
```

Comment:
why not directly `throw Exceptions.unwrap(e)` ?

Reply:
Because it returns a throwable and then we get Unhandled exception: java.lang.Throwable
So we would have to change the method's signature